Description: Fix guitests, may be unnecessary. I don't think upstream even runs them.
Author: Joshua Honeycutt <joshua.honeycutt@gmail.com>
Forwarded: no
Last-Update: 2016-11-21
--- a/src/test/guitests/guitests.pro
+++ b/src/test/guitests/guitests.pro
@@ -8,8 +8,7 @@
 INCLUDEPATH += ../../gui/src
 SOURCES += src/main.cpp \
     src/VersionCheckerTests.cpp
-HEADERS += src/VersionCheckerTests.h \
-    src/VersionChecker.h
+HEADERS += src/VersionCheckerTests.h
 win32 { 
     Debug:DESTDIR = ../../../bin/Debug
     Release:DESTDIR = ../../../bin/Release
--- a/src/test/guitests/src/VersionCheckerTests.cpp
+++ b/src/test/guitests/src/VersionCheckerTests.cpp
@@ -18,7 +18,7 @@
 
 #include "VersionCheckerTests.h"
 #include "VersionChecker.cpp"
-#include "../../gui/tmp/debug/moc_VersionChecker.cpp"
+#include "../../gui/tmp/release/moc_VersionChecker.cpp"
 
 #include <QtTest/QTest>
 
--- a/src/test/guitests/src/VersionCheckerTests.h
+++ b/src/test/guitests/src/VersionCheckerTests.h
@@ -18,7 +18,7 @@
 
 #pragma once
 
-#include "QObject.h"
+#include "qobject.h"
 
 class VersionCheckerTests : public QObject
 {
